modules/tinymce/src/plugins/table/main/ts/core/TableFormats.ts

```diff
@@ -11,39 +11,46 @@ const cellFormats = {
   tablecellbackgroundcolor: {
     selector: 'td,th',
     styles: { backgroundColor: '%value' },
-    remove_similar: true
+    remove_similar: true,
```

Comment:
There is a lot of duplication here now, can we remove some by making a function or "base" object to set the common items (e.g. selector, remove_similar and inherit)?